src/libs/relay/conduit_relay_io_handle.cpp

```diff
@@ -921,7 +871,14 @@ IOHandle::read(const std::string &path,
 {
     if(m_handle != NULL)
     {
-        m_handle->read(path, node);
+        if(path.empty())
+        {
+            m_handle->read(node);
+        }
+        else
+        {
+            m_handle->read(path, node);
+        }
     }
```

Comment:
resolves #570 , but we also need to add an extra unit test 

- [ ] add that test